db/migrate/20211025090827_devise_create_users.rb

```diff
@@ -22,10 +22,10 @@ def change
       # t.string   :last_sign_in_ip
 
       ## Confirmable
-      # t.string   :confirmation_token
-      # t.datetime :confirmed_at
-      # t.datetime :confirmation_sent_at
-      # t.string   :unconfirmed_email # Only if using reconfirmable
+      t.string   :confirmation_token
+      t.datetime :confirmed_at
+      t.datetime :confirmation_sent_at
+      t.string   :unconfirmed_email # Only if using reconfirmable
```

Comment:
Vispār nav labs standarts mainīt migrācijas, kuras jau ir bijušas master zarā. Katru migrāciju izpilda 1x un pieņesim, ka tev jau bija veikta piegāde un useri jau eksistē produkcijā. 

Rails tev vairāk šo migrāicju nepildīs - un ja pildītu, tad nofailotu,jo create_table :users nevarētu izpildīt, jo users tabula jau eksistē. Bet ja tu rollbackotu un atkārtoti izpildītu, tad tu pakāstu visus datus.

Nu karoč skaidrs, ka nav baigi izdevīgi labot jau izpil'ditas migrācijas, tāpēc labāk būtu bijis jauna migrācija ar `add_column :users, :confirma.....`

Tomēr mācību nolūkos šis ir fine, laidīšu cauri.